deps/patches/http_parser-win.patch

```diff
@@ -0,0 +1,59 @@
+diff --git a/Makefile b/Makefile
+index 5f4eb22..983f669 100644
+--- a/Makefile
++++ b/Makefile
+@@ -18,18 +18,30 @@
+ # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+ # IN THE SOFTWARE.
+ 
++VERSION = 2.7.0
+ PLATFORM ?= $(shell sh -c 'uname -s | tr "[A-Z]" "[a-z]"')
++ifeq ($(findstring cygwin,$(PLATFORM)), cygwin)
++PLATFORM = winnt
++endif
++ifeq ($(findstring mingw,$(PLATFORM)), mingw)
++PLATFORM = winnt
++endif
++
+ HELPER ?=
+ BINEXT ?=
+ ifeq (darwin,$(PLATFORM))
+-SONAME ?= libhttp_parser.2.7.0.dylib
++SONAME ?= libhttp_parser.$(VERSION).dylib
+ SOEXT ?= dylib
++else ifeq (winnt, $(PLATFORM))
++CC = gcc
++SONAME ?= libhttp_parser.$(VERSION).dll
```

Comment:
Either don't bother at all (since without symlinks it defeats much of the purpose), or do `libname-$major.dll`. Really we shouldn't need to patch this makefile though, I'd think we can set whatever we need as command line arguments.